DisguiseUnityRenderStream/Runtime/DisguiseRemoteParameters.cs

```diff
@@ -328,20 +328,25 @@ MemberInfo GetMemberInfo(UnityEditor.SerializedProperty property, out bool isPub
     }
 #endif
 
-    MemberInfo GetMemberInfoFromPropertyPath(string propertyPath)
+    MemberInfo GetMemberInfoFromPropertyPath(string propertyPath, bool ignoreCase = false)
     {
         // Note on fields/properties:
         // * BindingFlags.Public: DynamicSetterCache needs public access and to avoid exposing class internals
         // * BindingFlags.DeclaredOnly: Avoids System.Reflection.AmbiguousMatchException when a child class member
         //      overshadows a parent member (ex TextMeshPro.renderer). Since we traverse the hierarchy top to bottom,
         //      the top-level member will be resolved.
-        
+        // * BindingFlags.IgnoreCase: For secondary lookup (ex "Position" after "m_Position" failed we want to match both "Position" and "position").
```

Comment:
The root of these edge cases is that we're trying to match `UnityEditor.SerializedProperty.propertyPath` to a `System.Reflection.MemberInfo`.

This could be fixed in another PR where we wouldn't use `SerializedProperty` anymore, by enumerating fields/properties using `System.Serialization` directly.